src-vue/src/components/PullRequestsSelector.vue

```diff
@@ -0,0 +1,186 @@
+                    <el-button type="primary" @click="installLauncherPR(pull_request)">Install</el-button>
+                    <a target="_blank" :href="pull_request.html_url">
+                        {{ pull_request.number }}: {{ pull_request.title }}
+                    </a>
+                </el-card>
+            </el-collapse-item>
+
+            <el-collapse-item title="Mods PRs" name="2">
+                <div style="margin: 15px">
+                    <el-alert title="Warning" type="warning" :closable="false" show-icon>
+                        Mod PRs are installed into a separate profile. Make sure to launch via
+                        'r2ns-launch-mod-pr-version.bat' or via '-profile=R2Northstar-PR-test-managed-folder' to actually
+                        run the PR version!
+                    </el-alert>
+                </div>
+                <p v-if="pull_requests_mods.length === 0">
+                    <el-progress
+                        :show-text="false"
+                        :percentage="100"
+                        status="warning"
+                        :indeterminate="true"
+                        :duration="1"
+                        style="margin: 15px"
+                    />
+                </p>
+                <el-card v-else shadow="hover" v-for="pull_request in pull_requests_mods" v-bind:key="pull_request.url">
+                    <el-button type="primary" @click="installModsPR(pull_request)">Install</el-button>
+                    <a target="_blank" :href="pull_request.html_url">
+                        {{ pull_request.number }}: {{ pull_request.title }}
+                    </a>
+                </el-card>
+            </el-collapse-item>
+        </el-collapse>
+    </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue'
+import { PullRequestType } from '../../../src-tauri/bindings/PullRequestType';
+import { PullsApiResponseElement } from '../../../src-tauri/bindings/PullsApiResponseElement';
+import { invoke } from "@tauri-apps/api";
+import { ElNotification } from "element-plus";
+
+export default defineComponent({
+    name: 'PullRequestsSelector',
+    computed: {
+        pull_requests_launcher(): PullsApiResponseElement[] {
+            return this.$store.state.pull_requests_launcher;
+        },
+        pull_requests_mods(): PullsApiResponseElement[] {
+            return this.$store.state.pull_requests_mods;
+        },
+    },
+    methods: {
+        onChange(e: Object) {
+            const openedCollapseNames = Object.values(e);
+            if (openedCollapseNames.includes('1') && this.pull_requests_launcher.length === 0) {
+                this.getPullRequests('LAUNCHER');
+            }
+            if (openedCollapseNames.includes('2') && this.pull_requests_mods.length === 0) {
+                this.getPullRequests('MODS');
+            }
+        },
+        async getPullRequests(pull_request_type: PullRequestType) {
```

Comment:
I think that for more clarity, we should create a store module dedicated to PR installation with this method.

Reply:
So similar to the [`searchModule`](https://github.com/R2NorthstarTools/FlightCore/blob/78a9522e9ddae501b4ea1b4cd094a9a959b029f2/src-vue/src/plugins/modules/search.ts)?

Reply:
Oh nevermind you already wrote that above xD

Reply:
Ok, I moved to the store location to the submodule in 7d2b65b039a7bade3ce8478d03cc6b20846d6e34 but I'm not sure how to do move the methods as they are not `getters` (?) and not sure what property to use then... ^^"

Reply:
You can move methods there as `mutations`, and then summon them using `commit` calls.
Documentation here: https://vuex.vuejs.org/guide/mutations.html

Reply:
For moving `installLauncherPR` and `installModsPR` I'm not quite sure how to access `game_path` from the parent module store.

Reply:
You can import the `store` object and access its state:
```javascript
import {store} from "../store";
console.log(store.state.game_path);
```

Reply:
Done in 92d0d2009887a67353b2cee71b6c673fc6e5d7ca and 75f7deed46a9bc69da297060afed2cdc2e162737